Client/NoOpRunner.Client/Rendering/GameWindowRenderer.cs

```diff
@@ -19,6 +19,13 @@ public static void RenderMap(GamePlatforms platforms, Canvas canvas)
             var rectangleWidth = canvas.ActualWidth / platforms.SizeX;
             var rectangleHeight = canvas.ActualHeight / platforms.SizeY;
 
+            /* Use this code to paint different shapes in different colors
+            var enumerator = platforms.Shapes.GetEnumerator(); // Shapes
+            while (enumerator.MoveNext())
+            {
+                Type shapeType = enumerator.Current.GetType();
+            }
+            */
```

Comment:
Why use an explicit enumerator? Just `            foreach (var shape in platforms.Shapes)
            {
                var type = shape.GetType();
            }`

Reply:
Fixed